importer/downloader.py

```diff
@@ -5,19 +5,41 @@
 import sys
 from six.moves import urllib
 
-searchurl = "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&term=Retracted+Publication&retmax=10&retmode=json"
+searchurl = "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&term=Retracted+Publication&retmax=6000&retmode=json"
```

Comment:
Is there any way we can discover the number of articles available on PubMed here? Or does not specifying `retmax` just get you everything?

If so, maybe we can make this a CLI argument.